app/services/rapid_ready_for_decision/fast_track_pdf_generator.rb

```diff
@@ -19,22 +19,20 @@ class FastTrackPdfGenerator
       }
     }.freeze
 
-    def initialize(patient_info, blood_pressure_data, medications)
+    def initialize(patient_info, assessed_data, disability_type)
       @pdf = Prawn::Document.new
       @patient_info = patient_info
-      @blood_pressure_data = blood_pressure_data
-      @medications = medications
+      @blood_pressure_data = assessed_data[:bp_readings]
+      @medications = assessed_data[:medications]
       @date = Time.zone.today
+      @disability_type = disability_type
       @pdf.markup_options = PDF_MARKUP_SETTINGS
     end
 
     # progressively builds a pdf and is sensitive to sequence
     def generate
-      add_intro
-      add_blood_pressure_list
-      add_blood_pressure_outro
-      add_medications_list
-      add_about
+      template = File.read('app/services/rapid_ready_for_decision/views/hypertension.erb')
```

Comment:
You could use `@disability_type` here:
```suggestion
      template = File.read("app/services/rapid_ready_for_decision/views/#{@disability_type}.erb")
```